### DISCUSSION TITLE
Combining partitioned and unpartitioned assets

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#18264] [Q&A] [answered]
So far I handled everything by partition. (I got a certain set of input files and split them by hour)
I have another part of the flow which happens by element.

Is it possible to mix partitioned elements of the flow with non-partitioned ones?

**Comment by clairelin135** [comment, marked as answer]
Hi @yuval-airis.

Yes, it is possible for unpartitioned assets and partitioned assets to exist in the same job and to execute together.

Here's one example where an unpartitioned asset is downstream of a partitioned asset: https://github.com/dagster-io/dagster/discussions/15375
